convex_hull.cpp

```diff
@@ -184,22 +184,29 @@ int main(int argc, char* argv[]) {
 	long long int no_of_points;
 	if(scanf("%lld",&no_of_points) != 1)
 	{
-	  	fprintf(stderr, "Error, please enter a valid Integer.\n");
-	  	exit(1);
+		fprintf(stderr, "File is malformed. It should have two space separated real numbers per line.\n"
+			"First line denoting no. of points(N)\n"
+			"Following N lines containing two space separated real numbers denoting "
+			"x and y coordinates respectively.\n"
+			"Example :\n"
+			"2\n"
+			"1 1\n"
+			"2 2\n");
+		exit(1);
 	}
 	while(no_of_points--)
 	{
 		double xx, yy;
 
 		if(scanf("%lf",&xx) != 1)
 		{
-		  	fprintf(stderr, "Error, please enter a valid number.\n");
-		  	exit(1);
+			fprintf(stderr, "Error, please enter a valid number.\n");
```

Comment:
Iska kya?

Reply:
This was meant for cases like, missing input or values other than number. 
Example - 
`2`           
`1 A`
`1  `